libvirt/uri/ssh.go

```diff
@@ -53,17 +74,27 @@ func (u *ConnectionURI) parseAuthMethods() []ssh.AuthMethod {
 			agentClient := agent.NewClient(conn)
 			result = append(result, ssh.PublicKeysCallback(agentClient.Signers))
 		case "privkey":
-			sshKey, err := os.ReadFile(os.ExpandEnv(sshKeyPath))
-			if err != nil {
-				log.Printf("[ERROR] Failed to read ssh key: %v", err)
-				continue
-			}
+			for _,keypath := range sshKeyPaths {
```

Comment:
Was this run over go fmt? does it really allow no spaces after comma?

Reply:
I'm not totally clear what you're saying. Could you rephrase please?

Reply:
yes. I expect `_, keypath`. Please change. I am surprised go fmt does not change that by itself when you ran it?

Reply:
>yes. I expect _, keypath. Please change. I am surprised go fmt does not change that by itself when you ran it?

Truthfully, I am not a master of go. I was able to build and also run `go fmt` without any complaints just now.

Change is made.